src/components-examples/journey-maps/angular/journey-maps-full/journey-maps-full-example.html

```diff
@@ -20,7 +20,7 @@
     [journeyMapsZones]="form.get('zoneGeoJson')?.value"
     [poiOptions]="form.get('pointsOfInterestOptions')?.value"
     [(selectedMarkerId)]="selectedMarkerId"
-    [selectedLevel]="this.form.get('level')?.value ?? 0"
+    [selectedLevel]="this.form.get('level')?.value ?? undefined"
```

Comment:
```suggestion
    [selectedLevel]="this.form.get('level')?.value"
```

Nitpick, only because it's in the public example code.

Reply:
I fully agree. Just didn't notice it myself.